src/backup-all-indexes.sh

```diff
@@ -39,7 +55,17 @@ else
     REPOSITORY_PLUGIN=repository-s3
 fi
 
-backup_index ()
+
+# small slack notification function
+
+slack_send() { 
```

Comment:
Can you assign variables to locals to make `$1`, `$2` more explicit?

Reply:
`{` and `()` formatting is inconsistent with the rest of the file.